src/authorization/jwt/JwtIdentityProviderEnum.ts

```diff
@@ -11,4 +11,5 @@ export enum JwtIdentityProviderEnum {
   MAA_NoSecureBootTyFu = "https://maanosecureboottestyfu.eus.attest.azure.net",
   MAA_NoSecureBootWeu = "https://accnosecurebootmaawesteu.weu.attest.azure.net",
   MAA_NoSecureBootEus = "https://accnosecurebootmaa.eus2.attest.azure.net",
+  MAA_NoSecureBootInteTest = "https://confinfermaaeus2test.eus2.test.attest.azure.net",
```

Comment:
I have removed the necessity to add an issuer in code. This will not scale. I will sync my changes so we do no longer need this